### PR TITLE
br: add document for backup compression

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add comments about backup config: compression && compression-level

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/use-br-command-line-tool.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ br backup full --pd "${PD_IP}:2379" \
 * `--key`：指定 PEM 格式的 SSL 证书密钥文件路径。
 * `--status-addr`：向 Prometheus 提供统计数据的监听地址。
 * `--concurrency`：备份或恢复阶段的任务并发数。
+* `--compression`：备份生成文件的压缩算法，支持 lz4|snappy|zstd，默认 zstd(多数情况下无须修改)。
+* `--compression-level`：备份选择的压缩算法对应的压缩级别，zstd 默认 3(大多数情况下情况无须设置)。
```
Comment (BornChanger) [inline br/use-br-command-line-tool.md:52]: 这么写的话, 有没有资料可以告诉客户不同压缩算法+压缩级别的收益和代价, 有官方发的数据或者是我们自己测试的数据

Comment (3pointer) [reply br/use-br-command-line-tool.md:52]: 关于如何选择不同算法的解释可以加一个 rocksdb 官方的链接。https://github.com/EighteenZi/rocksdb_wiki/blob/master/Compression.md

File: br/use-br-command-line-tool.md [hunk 2]
```diff
@@ -48,6 +48,8 @@ br backup full --pd "${PD_IP}:2379" \
 * `--key`：指定 PEM 格式的 SSL 证书密钥文件路径。
 * `--status-addr`：向 Prometheus 提供统计数据的监听地址。
 * `--concurrency`：备份或恢复阶段的任务并发数。
+* `--compression`：备份生成文件的压缩算法，支持 lz4|snappy|zstd，默认 zstd(多数情况下无须修改), 如何选择不同的压缩算法可以参考[文档](https://github.com/EighteenZi/rocksdb_wiki/blob/master/Compression.md)。
```
Comment (ran-huang) [inline br/use-br-command-line-tool.md:51]:
```suggestion
* `--compression`：备份生成文件的压缩算法，支持 `lz4`、`snappy`、`zstd`，默认 `zstd`（多数情况下无须修改）。如何选择不同的压缩算法，可以参考[文档](https://github.com/EighteenZi/rocksdb_wiki/blob/master/Compression.md)。
```

File: br/use-br-command-line-tool.md [hunk 3]
```diff
@@ -48,6 +48,8 @@ br backup full --pd "${PD_IP}:2379" \
...
+* `--compression-level`：备份选择的压缩算法对应的压缩级别，zstd 默认 3(大多数情况下情况无须设置)。
```
Comment (ran-huang) [inline br/use-br-command-line-tool.md:52]:
```suggestion
* `--compression-level`：备份选择的压缩算法对应的压缩级别，`zstd` 默认为 3。大多数情况下无需设置。
```

**Review by ran-huang — APPROVED** [review]

**Review by Leavrth — APPROVED** [review]

**Pull request merged**